test/basic.html

```diff
@@ -262,13 +262,19 @@
 
     test('can submit with method=post', function(done) {
       form = fixture('FormPost');
-
+      var before_submit = false;
```

Comment:
Conventionally, we use `camelCase` for all identifiers. Please change this to use that form.